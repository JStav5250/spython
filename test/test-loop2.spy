for char i in "plt2024":
    print(i)
