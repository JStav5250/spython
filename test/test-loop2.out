p
l
t
2
0
2
4
